wordpress/schema.yaml

```diff
@@ -87,6 +87,10 @@ properties:
     - ClusterIP
     default: ClusterIP
     title: Service type of WordPress
+  maskedField:
```

Comment:
should we give this a more use-case-centric name, like `customSecret`?

Reply:
Updated.

I was considering such a name originally.
Also added fields like "title" since this would be shown in the UI.

/gcbrun